src/DIRAC/Interfaces/scripts/dsub.py

```diff
@@ -501,7 +502,7 @@ def main():
             errorList.append(str(e))
 
     for error in errorList:
-        print("ERROR %s: %s" % error)
+        gLogger.error("ERROR %s: %s" % error)
```

Comment:
`error` is just a single string (line 502), right? So is this correct?

Reply:
Yes, I just more or less blindly replaced the prints with gLoggers. I'll fix it.

Reply:
Fixed.